.travis.yml

```diff
@@ -30,8 +31,7 @@ addons:
 before_script:
   - createdb teletype-test
 
-# TODO: Once https://github.com/atom/ci/pull/79 is merged, update URL below to use build-package.sh on master branch
-script: 'curl -s https://raw.githubusercontent.com/atom/ci/a5ca10c32676334605be6492369327056e6692b0/build-package.sh | sh'
+script: 'curl -s https://raw.githubusercontent.com/atom/ci/master/build-package.sh | sh'
```

Comment:
Note that piping a script like this can lead to issues if the connection gets killed in the middle of the script executing and it can be _very_ hard to diagnose as the script output gets quite garbled. That's why `atom/ci` recommends [this way](https://github.com/atom/ci/blob/83744c0308ea7188ff90bd719a86b9dc0fb1d0d0/.travis.yml#L18-L21).

Reply:
@Arcanemagus: Thanks for the tip! https://github.com/atom/teletype/pull/304 adopts `atom/ci`'s recommended approach.